docs/index.md

```diff
@@ -38,41 +38,39 @@ resource "ec_deployment" "my_deployment" {
 
 ## Authentication
 
-The Elastic Cloud provider offers two methods of authentication against the remote API. Depending on the target environment, one or all can be used. By default the `endpoint` which the provider will target is the Public API of the Elasticsearch Service (ESS) offering.
+The Elastic Cloud Terraform provider offers two methods of authentication against the remote API: `apikey` or a combination of `username` and `password`. Depending on the environment, one or both can be used. The Public API of Elasticsearch Service (ESS) is the default `endpoint` that the provider will target.
```

Comment:
I do not think that both can be used,  either `username` and `password` or `apikey` can be used, but not both.

Reply:
From the original text, these sentences make think that ESS accepts only apikeys, while ECE accepts both, apikeys and username+pwd:

> "Depending on the target environment, one or all can be used."
> 
> "API Keys are the recommended authentication method. They can be used when authenticating against the Elasticsearch Service (ESS) or Elastic Cloud Enterprise (ECE)"
> 
> "When targeting an ECE Installation, username and password can be used."
>

Reply:
If you want, we can chat/clarify on slack later today when you get online?

Reply:
Ignore my comment then, I misread what you wrote, it looks good.